src/util/webauthn.js

```diff
@@ -0,0 +1,64 @@
+/* jshint es5:true */
```

Comment:
Based on the link here: https://github.com/adrianba/fido-snippets/blob/master/LICENSE.md the snippet code has MIT license.

@rchild-okta @mauriciocastillosilva-okta ^^